### PR TITLE
Fix humanize numbers

### PR DESCRIPTION
**Pull request by Adebesin-Cell**
# Fix humanize numbers

Added commas to make rank price numbers readable.

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/1716

### CODE REVIEW

**Comment by Tolu-Mals** [comment]
you should merge the update fix rank persistence branch to this

**Comment by Adebesin-Cell** [comment]
> you should merge the update fix rank persistence branch to this

Yeah, it's up to date with the fix rank persistence branch.

File: src/utils/DataTransform/getObjectValue.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  object: Record<string, T>,
+  value: T,
+): string | undefined =>
+  Object.keys(object).find((key) => object[key] === value)
+
+export type CategoryKey<T> = {
+  [K in keyof T]: K
+}[keyof T]
+
+export type CategoryKeyType = CategoryKey<typeof CATEGORIES_WITH_INDEX>
```
Comment (Tolu-Mals) [inline src/utils/DataTransform/getObjectValue.ts:13]: This is the old version of this file

Comment (Adebesin-Cell) [reply src/utils/DataTransform/getObjectValue.ts:13]: Seen, pushing a merge now.

File: src/types/RankDataTypes.ts [hunk 2]
```diff
@@ -78,3 +79,5 @@ export interface RankTableProps {
   currentPage?: number
   pageSize?: number
 }
+
+export type ObjectKeyType<T> = keyof T
```
Comment (Royal-lobster) [inline src/types/RankDataTypes.ts:83]: using just `keyof` would be simpler 😅

**Review by Royal-lobster — APPROVED** [review]

File: src/types/RankDataTypes.ts (hunk 2 above)
Comment (Adebesin-Cell) [reply src/types/RankDataTypes.ts:83]: I see 🫠

**Pull request merged**